azurerm/resource_arm_relay_namespace.go

```diff
@@ -57,6 +60,16 @@ func resourceArmRelayNamespace() *schema.Resource {
 				},
 			},
 
+			"sku_name": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ConflictsWith:    []string{"sku"},
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
As above, unless there is good reason should be case sensitive

Reply:
Fixed...